reddit_liveupdate/public/static/css/liveupdate-public.less

```diff
@@ -0,0 +1,123 @@
+// CSS in here is imported into reddit's main CSS file, for inclusion of live features into reddit proper
```

Comment:
Yeah, I think the benefit of sharing that class is marginal, but even if they weren't shared, for plugin separation reasons we'd still want a "liveupdate css that is injected into the main CSS" file.

You're right, a style tag -is- a crazy idea. ;) But, I'm willing to try it out and see how it feels. It definitely is a small number of bytes, I just worry about it being non-obvious for future devs. I'll try it!